_maps/map_catalogue.txt

```diff
@@ -573,3 +572,7 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		File name ="_maps\RandomRuins\wasteruins\wasteplanet_unhonorable.dmm"
 		Size = (x = 11)(y = 17)(z = 1)
 		Tags = "No Combat", "Medium Loot", "Shelter", "Hazardous"
+
+		File name = "_maps\RandomRuins\wasteruins\wasteplanet_abandoned_mechbay
+		Size = (x = 45)(y = 47)(z = 1)
+		Tags = "Boss Combat Challenge" "Medium Loot" "Shelter" "Hazardous"
```

Comment:
commas here

Reply:
This still needs commas. Between the tags.

Reply:
![image](https://user-images.githubusercontent.com/94164348/215960747-5068be38-a352-4d3e-8bba-216d7fdf9abb.png)

Reply:
I saw that and resolved it again, apologies.